program/src/processor.rs

```diff
@@ -1399,6 +1299,14 @@ impl Processor {
             .find_spot_market_index(spot_market_ai.key)
             .ok_or(throw_err!(MangoErrorCode::InvalidMarket))?;
 
+        // Don't allow initialization of spot open orders if market is in ForceClose or Swapping
+        // Don't need to check MarketMode::Inactive because spot market is zero key in that case
+        let mode = mango_group.tokens[market_index].spot_market_mode;
+        check!(
+            !(mode == MarketMode::ForceCloseOnly || mode == MarketMode::SwappingSpotMarket),
```

Comment:
possibly there could be `spot_market_mode.is_reduce_only()` and `spot_market_mode.allow_new_open_orders()`